javascript/node/selenium-webdriver/chromium.js

```diff
@@ -86,6 +86,8 @@ const WebSocket = require('ws')
 const cdp = require('./devtools/CDPConnection')
 const remote = require('./remote')
 const cdpTargets = ['page', 'browser']
+const fs = require('fs')
+let cdpCommandId = 0
```

Comment:
Either use [uuid](https://en.wikipedia.org/wiki/Universally_unique_identifier#Version_4_(random)), or place the integer on the driver, please don't keep it as a common running number for all connections.

Reply:
cdp API accepts `integer` as the value for `id`

So I have used a random number generator.